### PR TITLE
[WVD - MSIX App Attach] Adding new MSIX Package cmdlets and Updated Applications cmdlets for DesktopVirtualization 

### PR DESCRIPTION
**Pull request by shailahirji**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by wyunchi-ms** [comment]
Hi @shailahirji , the swagger is not ready. https://dev.azure.com/azure-sdk/internal/_build/results?buildId=571104&view=logs&j=c951caf4-2750-57cb-773f-b58d0651d461&t=485e42ac-f6ad-5197-11a7-c3f5710ca2b5&l=24. When will the swagger be ready?

**Comment by shailahirji** [comment]
Hi Yunchi,

I have reached out to the arm api approval team and have asked for their review as soon as possible. Is it possible to begin the powershell review with the in progress swagger PR?

Regards

Sheila

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by shailahirji** [comment]
Hi @wyunchi-ms , the PR has been approved and merged for swagger: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2020-09-21-preview

**Comment by shailahirji** [comment]
Hello again @wyunchi-ms , please lets try to resolve this but **not merge** it yet. I am waiting for another PR to come through for swagger before merging this

File: src/DesktopVirtualization/readme.md
Comment (datr93) [inline src/DesktopVirtualization/readme.md:56]: don't merge this until sso swagger and ps is merged

**Comment by datr93** [comment]
@dolauli can you take a look :)

File: src/DesktopVirtualization/custom/New-AzWvdMsixpackage_PackageAlias.ps1
Comment (wyunchi-ms) [inline src/DesktopVirtualization/custom/New-AzWvdMsixpackage_PackageAlias.ps1:1]: We don't use `_` in our cmdlet names. I suggest to rename it to `New-AzWvdMsixPackageAlias`.

File: src/Aks/readme.md
Comment (wyunchi-ms) [inline src/Aks/readme.md:73]: What's the reason of updating Aks? We are not using generated version of Aks but using hand writing one.

File: src/DesktopVirtualization/custom/New-AzWvdMsixpackage_PackageAlias.ps1
Comment (shailahirji) [reply src/DesktopVirtualization/custom/New-AzWvdMsixpackage_PackageAlias.ps1:1]: The '_' is actually not part of the cmdlet name. Does it still matter? this is the official cmdlet:
        $package_created = **New-AzWvdMsixPackage** -PackageAlias 'msixpackage' `
            -ImagePath 'C:\msix\singlemsix.vhd' `
            -HostPoolName $env.HostPool `
            -ResourceGroupName $env.ResourceGroup `
            -SubscriptionId $env.SubscriptionId  `
            -DisplayName 'package-Alias-test' `
            -IsActive 

I was just following the same convention as for the custom cmdlet for New-AzWvdApplication_AppAlias

File: src/Aks/readme.md
Comment (shailahirji) [reply src/Aks/readme.md:73]: Extract from an Email thread
---------------------------------------------------------------------------
From: Xiaogang Ding 
Sent: Wednesday, October 21, 2020 7:52 PM
To: Shaila Hirji ; Yunchi Wang; Yeming Liu ; Dingmeng Xue 
Cc: Karthik Kattamudi ; Nataraj Srikantaiah 
Subject: RE: Pipeline Check failure for Service:ResourceMover 

Hi Shaila,
 
Thanks for pointing that out. Resource Mover team moved the swagger location but did not change it in readme.md. I have changed it and retrigger the CI. 
 
Thanks,
Xiaogang
 
From: Shaila Hirji 
Sent: Thursday, October 22, 2020 5:40 AM
To: Xiaogang Ding ; Yunchi Wang; Yeming Liu; Dingmeng Xue 
Subject: Pipeline Check failure for Service:ResourceMover
 
Hi, 
 
I have been encountering this error : Error: Could not read 'https://github.com/Azure/azure-rest-api-specs/tree/master/specification/regionmove/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/regionmovecollection.json'. and not sure which file needs to be modified and to what. The failure seems to be while accessing some file in the azure-rest-api repo but the version may have been outdated. 
 
Here is the PR: https://github.com/Azure/azure-powershell/pull/13194
 
I did notice src/Aks/readme.md was pointing to something that was no longer part of the rest-api repo:  
 
So I changed it to this:  

But don’t think this resolved the error. Let me know if I should revert it back to the original one. 
 
Please assist with this as it is blocking us from making this available in selfhost. 
 
Thanks, 
 
Shaila

Comment (wyunchi-ms) [reply src/Aks/readme.md:73]: I think we need to revert it back to the original one.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**
